plotly/tests/test_optional/test_px/test_px_input.py

```diff
@@ -391,7 +399,12 @@ def test_build_df_with_hover_data_from_vaex_and_polars(test_lib, hover_data):
     )
 
 
-def test_timezones(constructor):
+def test_timezones(request, constructor):
+    # FIXME: PyArrow requires a different format for datetime constructors with timezones
+    from .conftest import pyarrow_table_constructor
+    if constructor is pyarrow_table_constructor:
+        request.applymarker(pytest.mark.xfail)
+
     df = nw.from_native(
         constructor({"date": ["2015-04-04 19:31:30+01:00"], "value": [3]})
```

Comment:
```suggestion
        constructor({"date": ["2015-04-04 19:31:30+0100"], "value": [3]})
```
@gvwilson here as well